src/components/input/SingleLineInputBase.js

```diff
@@ -79,6 +80,7 @@ class SingleLineInputBase extends PureComponent {
       onBlur: this.handleBlur,
       onFocus: this.handleFocus,
       readOnly,
+      innerRef,
```

Comment:
Strong PP: Implement ref forwarding similarly how it's done in other places, by using a forwardRef, so we can just pass `ref` to the component:

Example:
https://github.com/teamleadercrm/ui/blob/287612a6ffb295e628ede33fe41c7ef2a058457d/src/components/box/Box.js#L209

Reply:
Jep also like that :) will do